client/src/main/java/tds/assessment/Segment.java

```diff
@@ -143,7 +168,7 @@ public void setFieldTestMinItems(int fieldTestMinItems) {
     /**
      * @return the maximum number of field test items in the {@link tds.assessment.Segment}
      */
-    public int getFieldTestMaxItems() {
+    public Integer getFieldTestMaxItems() {
```

Comment:
Actually, as currently coded I don't think it is possible for them to be null because your using a primitive int to populate it ([here](https://github.com/SmarterApp/TDS_AssessmentService/pull/27/files#diff-855a89d888831240829f4e52cb2578cbR135)) and the autoboxing will not allow it to be null.

Same for all the field test ints.